packages/astro/src/pages/about/code-of-conduct.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: "@/layouts/BaseLayout.astro"
+title: Code of Conduct
+description: Code of Conduct
+output: html_document
```

Comment:
What does this prop do?  Don't think I've seen it before.

Reply:
This was actually a copy paste mistake while moving over some of the structure from other pages. There is an output mode which goes in your output config file to specify how your server will render the pages. But we don't need that here. I have removed this.